### ISSUE TITLE
Crash while menuing (sound) options

### ISSUE THREAD

**Issue by Endaris** [panel-attack/panel-game#274]
Upon menuing options on the way to or while adjust sound volume, reproduced on beta current at 2022-12-17 by Musichael.

```
Error

options.lua:1158: attempt to index upvalue 'memory_before_options_menu' (a nil value)


Traceback

[love "callbacks.lua"]:228: in function 'handler'
options.lua:1158: in function 'selectFunction'
click_menu.lua:257: in function 'selectButton'
click_menu.lua:379: in function 'click_or_tap'
main.lua:191: in function <main.lua:189>
[love "callbacks.lua"]:154: in function <[love "callbacks.lua"]:144>
[C]: in function 'xpcall'
```

**Comment by JamesVanBoxtel** [comment]
I can't reproduce this one nor can I see why it would break in the code. Recommend closing.

**Comment by Endaris** [comment]
Presumably fixed by panel-attack/panel-attack#873 
Closed as fixed until hearing opposite news.
